software/holoplay.yml

```diff
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
```

Comment:
```suggestion
description: "Save music to favorites and create playlists (alternative to Youtube)."
```

Reply:
I don't think `(alternative to Youtube)` is appropriate, it's just a web app that uses Invidious API to consume content from Youtube.